src/parser/shaman/shared/talents/EarthShield.tsx

```diff
@@ -18,8 +18,9 @@ import Combatants from 'parser/shared/modules/Combatants';
 import { Trans } from '@lingui/macro';
 
 import { HEALING_ABILITIES_AMPED_BY_EARTH_SHIELD } from 'parser/shaman/shared/constants';
+import { EMBRACE_OF_EARTH_RANKS } from 'parser/shaman/restoration/constants';
 
-const EARTHSHIELD_HEALING_INCREASE = 0.20; // TODO add conduit
+export const EARTHSHIELD_HEALING_INCREASE = 0.20;
 
 class EarthShield extends Analyzer {
```

Comment:
This is a core spell now right? Shouldn't it go under that and not talents?

Reply:
its a talent for 2 out of 3 specs

Reply:
thats stupid

Reply:
no u